openedx/adg/lms/webinars/tests/test_models.py

```diff
@@ -0,0 +1,42 @@
+"""
+Tests for all the models in webinars app.
+"""
+from datetime import timedelta
+from unittest.mock import Mock
+
+import pytest
+
+from openedx.adg.lms.webinars.constants import BANNER_MAX_SIZE
+from openedx.adg.lms.webinars.tests.factories import WebinarFactory
+
+
+@pytest.mark.django_db
+@pytest.fixture(name='webinar')
+def webinar_fixture():
+    """
+    Create webinar fixture to use as a parameter to other pytests or fixtures
+    """
+    return WebinarFactory()
```

Comment:
I guess we don't need this fixture or do we? We can directly use `WebinarFactory()` in tests.

Reply:
It is more neat than initializing the factory in every test case.